src/param-parsing.ts

```diff
@@ -10,3 +24,25 @@ export const parseDelta = (delta: string) => {
     throw new Error("cannot be called with a negative value");
   return deltaInSeconds;
 };
+
+const convertToTimestamp = (date: Dateish): number => {
```

Comment:
Even though this is fine, I would have:

1. Only supported ISO format and used something like [this](https://date-fns.org/v2.27.0/docs/parseISO)
2. Isn't there any `npm` package that implements this kind of logic already? :thinking:

Reply:
Date tries to parse the string as ISO first, that's why I sticked to it and recommended using ISO only(the date-only and the whole-string format specifically). I think if anyone wants to use other features of Date, it is up to them to use it or not, but Date works fine for both of the use cases I wanted to cover. 
I didn't find a package that allowed Date strings as well as timestamps, but I would be open to use one if there is one.

I'm open to discuss if this is the right API to chose though, should we only allow Date and Date strings maybe? Is there any valid usecase to use a timestamp? I mean, probably the way to go there would be to transform it on the user's side.
On top of that, the only valid use case I see for using a timestamp is that you are adding seconds to the previous block timestamp, in which case I think the setTimeInterval function might be the right one to use.

Reply:
Yeah, exactly. Maybe just `ISO 8601` and that would be it, way much simpler and with less cases to test. I'm not quite sure I would use the unix time as input tbh.

Reply:
Ok, will do

Reply:
Just to be clear, do you mean the whole string? or all of the versions of ISO 8601?

IMO, we should include the full version (2021-12-07T18:18:42+00:00), and the date only one (i.e. 2021-12-07). What do you think